back/psychoApp/models/template_test.py

```diff
@@ -2,7 +2,8 @@
 
 
 class TemplateTest(models.Model):
```

Comment:
Please add the meta with the spanish name

Reply:
sadly here to verbose and plural in spanish is the same